test/runtests.jl

```diff
@@ -630,6 +630,7 @@ end
         @test mod(1hr+3minute+5s, 24s) == 17s
         @test mod2pi(360°) === 0°           # 2pi is 360°
         @test mod2pi(0.5pi*u"m/dm") ≈ pi    # just testing the dimensionless fallback
+        @test modf(2.5rad) === (0.5, 2.0)
```

Comment:
It would be good to add a test with a dimensionless unit that isn’t equal to 1, and another test that checks that an error is thrown for a dimensionful quantity:
```suggestion
        @test modf(2.5rad) === (0.5, 2.0)
        @test modf(-250cm/m) === (-1//2, -2//1)
        @test_throws MethodError modf(1m)
```